config/serverless.security.search_ai_lake.yml

```diff
@@ -7,3 +7,6 @@ xpack.features.overrides:
   ### The following features are Security features hidden in Role management UI for this specific tier.
   securitySolutionTimeline.hidden: true
   securitySolutionNotes.hidden: true
+ 
+## Agentless deployment by default
+xpack.fleet.agentless.isDefault: true
```

Comment:
It seems that this configuration is not scoped to one integration, is it on purpose? What other Agentless integrations are available on Ai4DSoc projects? Asking because we are probably going to change their default deployment mode.

Reply:
@amirbenun 

> It seems that this configuration is not scoped to one integration, is it on purpose?

Yes

> What other Agentless integrations are available on Ai4DSoc projects? Asking because we are probably going to change their default deployment mode.

The new integrations page itself is still a WIP, but for the MVP there will be only [5 agentless integrations ](https://github.com/elastic/security-team/issues/11789#issuecomment-2672872286). I didn't want to update the integrations themselves to default to agentless deployments (ie [crowdstrike](https://github.com/elastic/security-team/issues/11789#issuecomment-2672872286)) since that would have ramifications outside the ai4dsoc project, so I added this config setting to the project type.